### DISCUSSION TITLE
weight_optimize not a valid argument for regressor

### DISCUSSION THREAD

**Discussion by krn-hov** [MilesCranmer/PySR#278] [Q&A]
When I try and set the weight_optimize parameter I get the 
TypeError: weight_optimize is not a valid keyword argument for PySRRegressor.
The regressor should update automatically I thought, in case this should be a version issue. But yes, I am curious if others are able to set this parameter. Especially since I want to use a large ncyclesperiteration.

**Comment by MilesCranmer** [comment]
Moved to issues: #279
